jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -150,7 +150,7 @@ class EntityDetails extends React.Component {
     @param {Array} models A list of the entity models found.
   */
   _fetchCallback(error, data) {
```

Comment:
Pre-existing: please remove the `@method` entry in this docstring: it's not required and it's wrong.